### PR TITLE
ipmitool: fix upstream

### PR DESCRIPTION
**Pull request by b-reich**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by b-reich** [comment]
I can fix this when codeberg.org is working again
https://status.codeberg.org/status/codeberg

**Comment by chenrui333** [comment]
looks like codeberg service is back to normal now.

File: Formula/ipmitool.rb [hunk 1]
```diff
@@ -27,20 +25,6 @@ class Ipmitool < Formula
...
-    sha256 "c7df82eeb6abf76439ca9012afdcef2e9e5ab5b44d4a80c58c7c5f2d8337bc83"
-  end
-
-  # Patch to fix build on ARM
-  # https://github.com/ipmitool/ipmitool/issues/332
-  patch do
-    url "https://github.com/ipmitool/ipmitool/commit/a45da6b4dde21a19e85fd87abbffe31ce9a8cbe6.patch?full_index=1"
-    sha256 "98787263c33fe11141a6b576d52f73127b223394c3d2c7b1640d4adc075f14d5"
-  end
-
```
Comment (chenrui333) [inline Formula/ipmitool.rb:43]: which patches removal, maybe bump the `revision` as well?

Comment (b-reich) [reply Formula/ipmitool.rb:43]: was work in progress, is now fixed

Comment (b-reich) [reply Formula/ipmitool.rb:43]: (and pushed when the tain wifi is working)

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
Thanks @b-reich!

**Review by iMichka — DISMISSED** [review]

**Pull request closed without merging**